plugins/techdocs/api-report.md

```diff
@@ -112,7 +112,17 @@ export const EmbeddedDocsRouter: (
 ) => JSX.Element | null;
 
 // @public
-export const EntityListDocsGrid: () => JSX.Element;
+export const EntityListDocsGrid: ({
+  group,
+}: EntityListDocsGridPageProps) => JSX.Element | null;
+
+// @public
+export type EntityListDocsGridPageProps = {
+  group?: {
```

Comment:
Hey @iamEAP, thanks for the initial review

I think that makes sense, though it ultimately depends on how we want `EntityListDocsGrid` to be used. I originally used it to invoke it individually for each group from my CustomHomePage component, but it also makes sense to invoke it once for all of the groups (the usage is probably easier?)

Anyway, I've made the change to use a list of groups as parameter rather than an individual group